pkg/aws_tags.go

```diff
@@ -128,6 +128,7 @@ func (iface tagsInterface) get(ctx context.Context, job *Job, region string) ([]
 	if len(svc.ResourceFilters) > 0 {
 		inputparams := &resourcegroupstaggingapi.GetResourcesInput{
 			ResourceTypeFilters: svc.ResourceFilters,
+			ResourcesPerPage:    aws.Int64(100), // max allowed value,
```

Comment:
From local testing seems that by default the API will return up to 50 entries per page.